<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmEquipment</class>
 <widget class="QWidget" name="frmEquipment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>68</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>68</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>68</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widgetMenu1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>231</width>
     <height>372</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget#widgetMenu1{background-image: url(:/images/treemenu/bg_menu1.png);}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetMenu2" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>231</width>
     <height>372</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget#widgetMenu2{background-image: url(:/images/treemenu/bg_menu2.png);}</string>
   </property>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>201</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>189</width>
       <height>329</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>120</width>
        <height>400</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetMenu3" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>231</width>
     <height>372</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget#widgetMenu3{background-image: url(:/images/treemenu/bg_menu3.png);}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetMenu4" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>70</y>
     <width>231</width>
     <height>372</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget#widgetMenu4{background-image: url(:/images/treemenu/bg_menu4.png);}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
